src/main/java/it/pagopa/pn/deliverypush/utils/HtmlSanitizer.java

```diff
@@ -0,0 +1,97 @@
+package it.pagopa.pn.deliverypush.utils;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.owasp.html.HtmlPolicyBuilder;
+import org.owasp.html.PolicyFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.util.CollectionUtils;
+
+import java.io.IOException;
+import java.util.*;
+
+/**
+ * Class that performs via the {@link #sanitize(Object)} method a cleanup of input parameters,
+ * allowing only determined HTML elements, based on the policies set in the constructor via the field {@link #policy}.
+ * <p>
+ * For example, if you wanted to allow only the HTML img element in the string, you could use:
+ * PolicyFactory policy = Sanitizers.IMAGES;
+ * or if you want no one HTML element: PolicyFactory policy = new HtmlPolicyBuilder().allowElements("").toFactory();
+ * <p>
+ * Example of usage with no HTML element policy:
+ * <p>
+ * String sanitized = policy.sanitize("<html><h1>SSRF WITH IMAGE POC</h1> <img src='https://prova.it'></img></html>");
+ * // sanitized value is "SSRF WITH IMAGE POC".
+ */
+@Component
+public class HtmlSanitizer {
+
+
+    private final ObjectMapper objectMapper;
+
+    private final PolicyFactory policy;
+
+    public HtmlSanitizer(ObjectMapper objectMapper) {
+        this.objectMapper = objectMapper;
+        this.policy = new HtmlPolicyBuilder().allowElements("").toFactory();
```

Comment:
Prima di poter approvare questa policy bisogna sapere con certezza che "&lt;h1&gt;" o stringhe simili non siano mai, per nessun motivo, una ragione sociale valida.

Reply:
Esternalizzato il comportamento della sanificazione in base alla property da application.property sanitize-mode, o da env variable SANITIZE_MODE (con priorità alla env variable). Valori possibili: ESCAPING e DELETE_HTML. Se non è presente né da env prop né da application.property, il valore di default è ESCAPING.